src/main/java/com/danielagapov/spawn/Services/FriendTag/IFriendTagService.java

```diff
@@ -13,4 +13,6 @@ public interface IFriendTagService {
     public FriendTagDTO replaceFriendTag(FriendTagDTO friendTag, UUID tagId);
     public boolean deleteFriendTagById(UUID id);
     public void saveUserToFriendTag(UUID id, UUID userId);
+
+    void deleteUserFromFriendTag(UUID everyoneTagId, UUID friendId);
```

Comment:
```suggestion
    public void deleteUserFromFriendTag(UUID id, UUID userId);
```
Correct me if I'm wrong, but it seemed strange how this method signature didn't match what's in `FriendTagService`, so I adjusted it